exercises/concept/assembly-line/tests/assembly-line.rs

```diff
@@ -1,5 +1,7 @@
 fn process_rate_per_hour(speed: u8, expected_rate: f64) {
-    assert!((assembly_line::production_rate_per_hour(speed) - expected_rate).abs() < f64::EPSILON);
+    let actual_rate = assembly_line::production_rate_per_hour(speed);
+    let actual_rate = (actual_rate * 100.0).floor()/100.0;
```

Comment:
```suggestion
    let actual_rate = (actual_rate * 100.0).round()/100.0;
```

If the introduced floating-point error happens to be low instead of high, we'll have more accurate results by rounding instead of flooring.